db/migrate/20230209211722_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string
```

Comment:
- [ ] Consider adding a default value for the user role as a user so that in a situation where the role is not explicitly defined, it defaults to user. You could do so like this: ```    add_column :users, :role, :string, default: "user"```